platform/engineering/frontend/eslint-additional-rules.md

```diff
@@ -0,0 +1,73 @@
+# ESLint Additional Rules
+
+The front-end team will be introducing a new set of rules from a third-party plugin called SonarJS ([eslint-plugin-sonarjs](https://github.com/SonarSource/eslint-plugin-sonarjs)).
+
+SonarJS delivers a static code analyzer for JavaScript. It uses the most advanced techniques (pattern matching, dataflow analysis) to analyze code and find code smells, bugs and security vulnerabilities. SonarJS has a great coverage of well-established quality standards.
+
+SonarJS rules for ESLint to detect bugs and suspicious patterns in the code can be found [here](https://github.com/SonarSource/eslint-plugin-sonarjs).
+
+## Rules Deployment
+
+The front-end team will be deploying SonarJS’ 25 rules in 3 different batches.
```

Comment:
Probably be worth calling out why there are three batches and what the differences between them are.